### PR TITLE
lesson_14

### PR DESCRIPTION
**Pull request by akpoik**
Signed-off-by: Aleksander Kulon <olo3@poczta.onet.pl>
@gradopaw

### CODE REVIEW

**Comment by akpoik** [comment]
@gradopaw

File: students/kulon_aleksander/lesson_14/authorization.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+access_count = -1
+
+
+def has_access():
+    '''
```
Comment (gradopaw) [inline students/kulon_aleksander/lesson_14/authorization.py]: You should use triple double-quoted for docstrings.

File: students/kulon_aleksander/lesson_14/decorator_logger_wrapper.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+from lesson_06.args_inspector import args_inspect
+
+
+def wrap_logs(foo, *args, **kwargs):
+    args_inspect(*args, **kwargs)
+    return foo(*args, *kwargs)
```
Comment (gradopaw) [inline students/kulon_aleksander/lesson_14/decorator_logger_wrapper.py]: ```return foo(*args, **kwargs)```

**Comment by akpoik** [comment]
@gradopaw

**Pull request merged**